File: python_programming/practices/old_enough.py
# LL 6th Old Enough

age = int(input("Enter your age: "))

if age >=21:
    print("your old enough to drink the fun juice")
elif age >= 18:
    print("You are old enough to vote, but not drink the fun juice")
elif age >= 16:
    print("You are old enough to drive, but not old enough to vote.")
elif age >= 15:
    print("You are old enough to get a learner's permit, but not a drivers license nor vote.")
elif age >= 5:
    print("You are old enough to go to school, but not old enough to have a learners permit, vote or drive")
else:
    print("You are not old enough for school, a permit, driving, or voting.")

File: c_programming/Practices/old_enough.c



#include <stdio.h>

int main() {
    int age;
    printf("How old are you?: ");
    scanf("%d", &age);
    
    if (age >= 21) {
        printf("You can drink the fun juice :)\n");
    } else if (age >= 18) {
        printf("You can vote, tokyo drift, get a permit, and go to school but you cant drink the fun juice :(.\n");
    } else if (age >= 16) {
        printf("You can tokyo drift and go to school.\n");
    } else if (age >= 15) {
        printf("You can get a permit and go to school.\n");
    } else if (age >= 5) {
        printf("You can go to school.\n");
    } else {
        printf("Stand ready for my arrival, worm.\n");
    }

    return 0;
}




